src/include/sof/audio/component.h

```diff
@@ -249,7 +249,8 @@ enum {
 		  (uint32_t)((pcd)->cpu_delta_peak))
 
 #define comp_perf_avg_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u",\
+	comp_info(comp_p, "perf component: %x samples %u period %u cpu avg %u peak %u",\
```

Comment:
Any reason we cant use core, id and pipeline id ?
```
struct comp_ipc_config {
	uint32_t core;		/**< core we run on */
	uint32_t id;		/**< component id */
	uint32_t pipeline_id;	/**< component pipeline id */
```

Reply:
The legacy sof-logger is limited to four 32bits arguments.

Reply:
Thanks, Liam and Marc.

The purpose is use this ID to find the module name:
uint32_t id; /**< component id */
Take volume as example, above id printed as: 0x60000, or 0x60001, I can't find it is volume module based on this.

(comp_p)->drv->uid->a
This is printed as: 0x61bca9a8, easy to find it is volume module uid.

However, I forgot to add core and pipeline id information, new commit is adding this.

Reply:
ipc4 perf log looks like this based on current commit:
component: comp:1 0x60001 perf c0 comp 61bca9a8 samples 48 period 1000 cpu avg 381 peak 403
1 --> pipe ID, 0x60001 -->  ipc_config.id, 61bca9a8->component ID

Reply:
> component: comp:1 0x60001 perf c0 comp 61bca9a8 samples 48 period 1000 cpu avg 381 peak 403
1 --> pipe ID, 0x60001 --> ipc_config.id, 61bca9a8->component ID

Shouldn't this be easier to parse? Maybe `comp:61bca9a8 samples:48 period:1000 cpu_avg:381 peak:403`

Reply:
comp 61bca9a8 samples 48 cpu avg 353 peak 372
yes, now it looks like this, I removed period, since 4 int request, also don't have :, if you want I can add.